### DISCUSSION TITLE
/etc/os-release 文件内容错误

### DISCUSSION THREAD

**Discussion by zhangguanzhang** [immortalwrt/immortalwrt#569] [Q&A] [answered]
https://github.com/immortalwrt/immortalwrt/blob/669bb7ac3013ccd592868eb9f0eca9a22f487cb3/package/base-files/files/usr/lib/os-release#L4
这行不改掉吗
我现在写个脚本判断当前固件是lede的还是immortalwrt的，哪个文件或者条件能作为判断依据的

**Comment by 1715173329** [comment, marked as answer]
- /etc/openwrt_release

   `DISTRIB_ID='ImmortalWrt'`

- /usr/lib/os-release

   `NAME="ImmortalWrt"`
   `ID="immortalwrt"`
